### PR TITLE
Typing events can be responded to like any other message

### PR DESCRIPTION
**Pull request by VxJasonxV**
This change was spurred by 'drewajp':

screen shot 2016-11-14 at 12 56 07 am

A reference code example:

```ruby
require 'discordrb'

bot = Discordrb::Bot.new token: ENV['DISCORD_TOKEN'], client_id: ENV['DISCORD_APPLICATION_ID']

bot.typing do |event|
  if event.user.id == ENV['DISCORD_ADMIN_USER_ID'].to_i
    event.respond %Q{You started typing at #{event.timestamp}}
  end
end

bot.run
```

When running this, the bot sent

```plain
You started typing at 2016-11-14 07:49:59 +0000
```

In the channel I started typing in. Tested it on two different Discord servers, one channel named `#general`, one not.

### CODE REVIEW

**Comment by beezTEM** [comment]
We need this because how many times do you think this might be helpful?

File: lib/discordrb/events/typing.rb [hunk 1]
```diff
@@ -5,6 +5,7 @@
 module Discordrb::Events
   # Event raised when a user starts typing
   class TypingEvent < Event
+    include Respondable
```
Comment (meew0) [inline lib/discordrb/events/typing.rb]: Please add a newline after this; while RuboCop doesn't require it I think it's good style to separate the included modules like that.

Comment (VxJasonxV) [inline lib/discordrb/events/typing.rb]: screen shot 2016-11-14 at 04 35 01

;D

**Comment by VxJasonxV** [comment]
Fixed not only my addition, but the two other locations where it was missing a newline too.

**Comment by meew0** [comment]
I didn't even know it was like that everywhere, thanks for finding that out!

**Pull request merged**